detox/android/detox/src/main/java/com/wix/detox/DetoxManager.java

```diff
@@ -56,9 +55,6 @@ public String invoke(Throwable t) {
 
         Bundle arguments = InstrumentationRegistry.getArguments();
         detoxServerUrl = arguments.getString(DETOX_SERVER_ARG_KEY);
-        if (detoxServerUrl != null) {
-            detoxServerUrl = detoxServerUrl.replace(Environment.DEVICE_LOCALHOST, Environment.getServerHost());
-        }
```

Comment:
Not needed at all thanks to `adb reverse` -- regardless of Genymotion. It's here now simply because genymotion has surfaced issues with it; getting rid of this logic solves it across the board.

Reply:
Makes sense.